### ISSUE TITLE
Two different bugs on different chromosome with same scripts

### ISSUE THREAD

**Issue by Dolce-99** [ghm17/LOGODetect#30]
Hi,

Thanks for providing such great work. However I ran into two bugs on different chromosomes when running same scripts. The first one seems to be related to node, however I can't specify any parameter since I'm working on server. The second one is related to heritability, but I don't know which part was wrong. Could you please help me look into what's happening here? Please see logs attached and let me know if you need more info. Thank you very much in advance!

Best

**Comment by ghm17** [comment]
Could you also provide the scripts that encounter these bugs?

**Comment by Dolce-99** [comment]
No problem, please see attached. For this scripts, some chromosomes successfully completed while some others encountered these bugs. Hope this info helps and great thanks again!
[05.run.logodetect.txt](https://github.com/user-attachments/files/18223001/05.run.logodetect.txt)

**Comment by ghm17** [comment]
It seems that LDSC did not successfully provide the heritability and genetic correlation estimates for these two chromosomes. Can you find those two ldsc_rg.log files under directory ./tmp_files/ldsc and send them to me for a look?

**Comment by Dolce-99** [comment]
Please see attached. Thanks a lot!

[ldsc_rg_bug1.log](https://github.com/user-attachments/files/18223627/ldsc_rg_bug1.log)
[ldsc_rg_bug2.log](https://github.com/user-attachments/files/18223628/ldsc_rg_bug2.log)

**Comment by ghm17** [comment]
Thank you for sharing the files. The heritability estimated by LDSC was negative for chromosome 6 and NaN for chromosome 8. This issue arises from the small number of SNPs within individual chromosomes, leading to unstable heritability estimates. Could you try running LOGODetect with genome-wide summary statistics instead? Using genome-wide SNPs will allow LDSC to generate more robust heritability estimates. You can still specify --chr CHR in LOGODetect for chromosome-specific analyses if needed.

**Comment by Dolce-99** [comment]
Thanks a lot! I tried inputing genome-wide summary statistics without specifying --chr. However I still encountered the Error in checkForRemoteErrors(val). I thought there were an issue also talking about this error while he fixed by specifying parameters of the clusters. However I'm working jobs on server where I can't specify anything. Do you have any suggestion for this? Also see log attached. Thank you again and happy holiday!
[new.log](https://github.com/user-attachments/files/18264894/new.log)

**Comment by ghm17** [comment]
It seems that for the first GWAS, the Mean chi^2 is only 1.002 and LDSC does not return a reasonable heritability estimate within (0, 1). Can you send me the two ldsc_rg.log files under directory ./tmp_files/ldsc for a further check?

**Comment by Dolce-99** [comment]
Right, the heritability was negative. Does this cause the sfLapply error?
[ldsc_rg.log](https://github.com/user-attachments/files/18294319/ldsc_rg.log)

**Comment by ghm17** [comment]
Yes, LOGODetect relies on positive heritability to construct the null distribution of the scan statistic. If heritability is negative, it results in taking the square root of a negative value, which causes an error. Furthermore, negative or near-zero heritability suggests that either the trait is not heritable or the GWAS is severely underpowered. Therefore, in such cases, it is unlikely that LOGODetect will identify any regions enriched for local genetic correlation.

**Comment by Dolce-99** [comment]
Okay I see. I've changed to another proper summary statistics and the software works well. Great thanks again for helping out! Really appreciated!
